### PR TITLE
updating requirements

### PR DESCRIPTION
**Pull request by AndrewLitteken**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-cirq>=0.14.0
-general-superstaq~=0.2.1
+cirq~=1.0.0
```
Comment (richrines1) [inline requirements.txt:1]: do we need cirq 1.0.0? usually we try to support a version or two back

Comment (AndrewLitteken) [reply requirements.txt:1]: Maybe not, but it's what the checks were requiring

Comment (richrines1) [reply requirements.txt:1]: ah that sounds like a bug in the check script - filed https://github.com/SupertechLabs/general-superstaq/issues/130

(i think it should still pass without the change though - i believe the cirq/qiskit version checks are just warnings)

**Review by richrines1 — APPROVED** [review]
lg (though i think this would also be closed by merging #319)

**Comment by AndrewLitteken** [comment]
True, but probably a better idea to keep unrelated fixes separate

**Pull request merged**